<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>914</width>
    <height>823</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>多线程IDM</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>891</width>
      <height>801</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0">
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="font">
        <font>
         <family>宋体</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>ArrowCursor</cursorShape>
       </property>
       <property name="mouseTracking">
        <bool>false</bool>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="tabPosition">
        <enum>QTabWidget::North</enum>
       </property>
       <property name="tabShape">
        <enum>QTabWidget::Triangular</enum>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="elideMode">
        <enum>Qt::ElideLeft</enum>
       </property>
       <property name="movable">
        <bool>true</bool>
       </property>
       <property name="tabBarAutoHide">
        <bool>false</bool>
       </property>
       <widget class="QWidget" name="tab">
        <attribute name="icon">
         <iconset>
          <normaloff>C:/Users/.designer/backup/下载.jpg</normaloff>C:/Users/.designer/backup/下载.jpg</iconset>
        </attribute>
        <attribute name="title">
         <string>下载</string>
        </attribute>
        <widget class="QWidget" name="layoutWidget">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>861</width>
           <height>751</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3" stretch="1,0,0,0">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1,1,1">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,1,9">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_5">
                <item>
                 <widget class="QLabel" name="label">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                    <horstretch>1</horstretch>
                    <verstretch>1</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="font">
                   <font>
                    <family>宋体</family>
                    <pointsize>14</pointsize>
                   </font>
                  </property>
                  <property name="text">
                   <string>下载类型：</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <widget class="QComboBox" name="DownloadType">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                  <horstretch>1</horstretch>
                  <verstretch>1</verstretch>
                 </sizepolicy>
                </property>
                <property name="currentText">
                 <string>URL</string>
                </property>
                <item>
                 <property name="text">
                  <string>URL</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>BVid</string>
                 </property>
                </item>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="URLlineEdit">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                  <horstretch>5</horstretch>
                  <verstretch>1</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <family>宋体</family>
                  <pointsize>10</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>https://dldir1.qq.com/music/clntupate/QQMusicSetup.exe</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="2,8">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_10" stretch="1,1">
                <item>
                 <widget class="QLabel" name="label_5">
                  <property name="enabled">
                   <bool>true</bool>
                  </property>
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                    <horstretch>1</horstretch>
                    <verstretch>1</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="font">
                   <font>
                    <family>宋体</family>
                    <pointsize>14</pointsize>
                   </font>
                  </property>
                  <property name="text">
                   <string>保存位置：</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="chooseLocation">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                    <horstretch>1</horstretch>
                    <verstretch>1</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="minimumSize">
                   <size>
                    <width>0</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="font">
                   <font>
                    <family>宋体</family>
                    <pointsize>12</pointsize>
                   </font>
                  </property>
                  <property name="text">
                   <string>选择目录</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEdit">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                  <horstretch>5</horstretch>
                  <verstretch>1</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <family>宋体</family>
                  <pointsize>14</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>C:/</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_8">
              <item>
               <widget class="QLabel" name="label_2">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="font">
                 <font>
                  <family>宋体</family>
                  <pointsize>14</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>线程数：</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSlider" name="horizontalSlider">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                  <horstretch>10</horstretch>
                  <verstretch>1</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimum">
                 <number>2</number>
                </property>
                <property name="maximum">
                 <number>16</number>
                </property>
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_7">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                  <horstretch>0</horstretch>
                  <verstretch>1</verstretch>
                 </sizepolicy>
                </property>
                <property name="font">
                 <font>
                  <family>宋体</family>
                  <pointsize>12</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>2</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_9">
              <item>
               <widget class="QLabel" name="label_3">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                  <horstretch>1</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="font">
                 <font>
                  <family>宋体</family>
                  <pointsize>14</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>下载进度：</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QProgressBar" name="MainprogressBar">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                  <horstretch>10</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <family>宋体</family>
                  <pointsize>12</pointsize>
                 </font>
                </property>
                <property name="value">
                 <number>0</number>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,1,2,1,2">
            <property name="spacing">
             <number>0</number>
            </property>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="startButton">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>5</horstretch>
                <verstretch>1</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>10</width>
                <height>0</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>宋体</family>
                <pointsize>14</pointsize>
               </font>
              </property>
              <property name="text">
               <string>开始下载</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="stopButton">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>停止下载</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QLabel" name="label_6">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>1</horstretch>
              <verstretch>1</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <family>宋体</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>下载详细信息</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPlainTextEdit" name="Download_info">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>6</horstretch>
              <verstretch>10</verstretch>
             </sizepolicy>
            </property>
            <property name="palette">
             <palette>
              <active>
               <colorrole role="Text">
                <brush brushstyle="SolidPattern">
                 <color alpha="255">
                  <red>200</red>
                  <green>13</green>
                  <blue>13</blue>
                 </color>
                </brush>
               </colorrole>
               <colorrole role="PlaceholderText">
                <brush brushstyle="SolidPattern">
                 <color alpha="128">
                  <red>200</red>
                  <green>13</green>
                  <blue>13</blue>
                 </color>
                </brush>
               </colorrole>
              </active>
              <inactive>
               <colorrole role="Text">
                <brush brushstyle="SolidPattern">
                 <color alpha="255">
                  <red>200</red>
                  <green>13</green>
                  <blue>13</blue>
                 </color>
                </brush>
               </colorrole>
               <colorrole role="PlaceholderText">
                <brush brushstyle="SolidPattern">
                 <color alpha="128">
                  <red>200</red>
                  <green>13</green>
                  <blue>13</blue>
                 </color>
                </brush>
               </colorrole>
              </inactive>
              <disabled>
               <colorrole role="Text">
                <brush brushstyle="SolidPattern">
                 <color alpha="255">
                  <red>120</red>
                  <green>120</green>
                  <blue>120</blue>
                 </color>
                </brush>
               </colorrole>
               <colorrole role="PlaceholderText">
                <brush brushstyle="SolidPattern">
                 <color alpha="128">
                  <red>0</red>
                  <green>0</green>
                  <blue>0</blue>
                 </color>
                </brush>
               </colorrole>
              </disabled>
             </palette>
            </property>
            <property name="font">
             <font>
              <family>Consolas</family>
              <pointsize>16</pointsize>
              <weight>9</weight>
              <italic>false</italic>
              <bold>false</bold>
             </font>
            </property>
            <property name="autoFillBackground">
             <bool>true</bool>
            </property>
            <property name="styleSheet">
             <string notr="true">font: 75 16pt &quot;Consolas&quot;;rgb:(255, 0, 0)</string>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
            <property name="plainText">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QWidget" name="tab_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <attribute name="icon">
         <iconset>
          <normaloff>C:/Users/.designer/backup/设置.jpg</normaloff>C:/Users/.designer/backup/设置.jpg</iconset>
        </attribute>
        <attribute name="title">
         <string>设置</string>
        </attribute>
        <widget class="QLabel" name="label_4">
         <property name="geometry">
          <rect>
           <x>60</x>
           <y>70</y>
           <width>181</width>
           <height>31</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>宋体</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>文件下载目录：</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_2">
         <property name="geometry">
          <rect>
           <x>210</x>
           <y>70</y>
           <width>331</width>
           <height>31</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>宋体</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>C:/</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton">
         <property name="geometry">
          <rect>
           <x>540</x>
           <y>70</y>
           <width>41</width>
           <height>31</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>宋体</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>...</string>
         </property>
        </widget>
        <widget class="QComboBox" name="Limit_Speed">
         <property name="geometry">
          <rect>
           <x>160</x>
           <y>180</y>
           <width>121</width>
           <height>21</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>宋体</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <item>
          <property name="text">
           <string>128 KB/s</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>256KB/s</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>512KB/s</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>1 M/s</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2M/s</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>5M/s</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>10M/s</string>
          </property>
         </item>
        </widget>
        <widget class="QCheckBox" name="checkBox">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>30</y>
           <width>171</width>
           <height>21</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>宋体</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>默认路径下载</string>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
        <widget class="QCheckBox" name="checkBox_2">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>130</y>
           <width>141</width>
           <height>31</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>宋体</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>不限速下载</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QCheckBox" name="checkBox_3">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>180</y>
           <width>121</width>
           <height>21</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>宋体</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>限速下载</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="tab_3">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <attribute name="icon">
         <iconset>
          <normaloff>C:/Users/.designer/backup/日志.jpg</normaloff>C:/Users/.designer/backup/日志.jpg</iconset>
        </attribute>
        <attribute name="title">
         <string>日志</string>
        </attribute>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>horizontalSlider</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>label_7</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>446</x>
     <y>127</y>
    </hint>
    <hint type="destinationlabel">
     <x>869</x>
     <y>127</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
